src/main/scala/cc/factorie/app/nlp/ner/OntonotesChainNer.scala

```diff
@@ -190,16 +190,6 @@ class BasicOntonotesNER extends DocumentAnnotator {
       if (word.length > 5) { features += "P="+cc.factorie.app.strings.prefix(word, 4); features += "S="+cc.factorie.app.strings.suffix(word, 4) }
       if (token.isPunctuation) features += "PUNCTUATION"
       if (lexicon.NumberWords.containsLemmatizedWord(word)) features += "#WORD"
-      if (lexicon.iesl.Money.containsLemmatizedWord(word)) features += "MONEY"
```

Comment:
They were removed as lines 208-217 are the correct way of using the new lexicons. It looks like these lines got added back in in a merge at some point as I removed them when I did the original lexicon pull request.